mage_ai/data_preparation/repo_manager.py

```diff
@@ -68,6 +68,7 @@ def __init__(self, repo_path: str = None, config_dict: Dict = None):
             self.gcp_cloud_run_config = repo_config.get('gcp_cloud_run_config')
             self.notification_config = repo_config.get('notification_config', dict())
             self.queue_config = repo_config.get('queue_config', dict())
+            self.sync_config = repo_config.get('sync_config', dict())
```

Comment:
Do you put the sync_config in metadata.yaml or preferences.yaml?

Reply:
oops let me delete this